<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar:vertical {
      background-color: transparent;
      width: 10px;
      margin: 1px 5 5px 0;
  }
  QScrollBar::handle:vertical {
      background: #bdc3c7;
	  min-height: 35px;
	  max-height: 50px;
	  border-radius: 2px;
  }
  QScrollBar::add-line:vertical {
		background-color: transparent;
		border: none;
		outline: none;
		
  }

  QScrollBar::sub-line:vertical {
      background-color: transparent;
		border: none;
		outline: none;

  }
  QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     background-color: transparent;
		border: none;
		outline: none;
  }

  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>119</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border: 1px solid #3498DB;
	min-width: 50px;
	padding: 3px;
	background-color: #258FD6;
	color: #FFEBEE;
	font: 8pt &quot;Open Sans&quot;;
}

QPushButton::hover{
	background-color: #3498db;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Background_border">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>1</y>
     <width>329</width>
     <height>149</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_Background_border{
background-color: rgba(21, 30, 38, 190); /*#3498DB #E2EAEB  (46, 64, 82, 245)*/;
border-radius: 1px;
}
/*border-top-left-radius: 5px;
border-top-right-radius: 5px;
border-bottom-right-radius: 3px;
border-bottom-left-radius: 3px;*/</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="bottomBorder">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>115</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F0F0F0;

</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QWidget" name="widget_TopMacButtons" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>5</y>
     <width>41</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_Exit_2">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	image: url(:/Exit/exit_mac_os);
}
QPushButton:hover{
	image: url(:/Exit/exit_hover_mac_os);
}</string>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>18</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Reduce_2">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	image: url(:/Reduce/reduce_mac_os);
}
QPushButton:hover{
	image: url(:/Reduce/reduce_hover_mac_os);
}</string>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="iconSize">
       <size>
        <width>28</width>
        <height>18</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>1</y>
     <width>26</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>26</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>26</width>
     <height>18</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>26</width>
     <height>18</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border: none;
	image: url(:/Exit/exit_windows_os);
}
QPushButton:hover{
	image: url(:/Exit/exit_hover_windows_os);
}</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>22</y>
     <width>321</width>
     <height>93</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
background-color: white;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Open Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>2</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 9pt &quot;Open Sans&quot;;
color: #3498db;
</string>
   </property>
   <property name="text">
    <string notr="true">vPicture</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_Background_border</zorder>
  <zorder>bottomBorder</zorder>
  <zorder>widget_TopMacButtons</zorder>
  <zorder>pushButton_Exit</zorder>
  <zorder>buttonBox</zorder>
  <zorder>textEdit</zorder>
  <zorder>label_title</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
